NuGet.config

```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-volatile-dev/api/v3/index.json" />
```

Comment:
No, it's fine to change this locally to make sure that your changes catch everything but we shouldn't check these changes in. Scaffolding will be broken until Coherence passes but that shouldn't take long.